### ISSUE TITLE
python 3 support

### ISSUE THREAD

**Issue by Sispheor** [bovender/indicator-ip#16]
As the end of python 2 is near, would be nie to convert this app to python 3.

**Comment by bovender** [comment]
Yeah, I noticed that when I set up a new virtual machine with Ubuntu 19.04 to develop this applet. (My main machine is a KDE neon system.) However, I'm really not very experienced at all with Python GUI programming. If someone (you...?!) could assist with upgrading this project to Python 3, that would be great.

Kind regards

Daniel

**Comment by Sispheor** [comment]
I sent a PR for this.

**Comment by bovender** [comment]
Anyone coming here looking for Python3 support should look at Sispheor's remake of this applet:

https://github.com/Sispheor/indicator-ip

See also: https://github.com/bovender/indicator-ip/pull/17#issuecomment-487504130
